src/CommandLine.cpp

```diff
@@ -133,7 +133,7 @@ void sub_612910(std::int32_t argc, char **argv) {
             for (std::uint32_t j = 0; j < y->count; ++j) {
                 y->args[j] = MC2_STRDUP(argv[i + j]);
             }
-            i = x;
+            i = x - 1;
```

Comment:
I remember debating to myself which way this should be. IIRC, I chose `i = x` because `x` acts like an end iterator (as in it points to the element *past* the last element), and so the next starting point should be `x`. I could be wrong (and I don't trust that the original code is right), but I'm not sure either way on this.